recipe/meta.yaml

```diff
@@ -24,15 +24,13 @@ requirements:
   run:
     - python >=3.8
     - typer >=0.12.3
-    - fastapi
-    - uvicorn >=0.15.0
+    - uvicorn-standard >=0.15.0
 
 test:
   imports:
     - fastapi_cli
   commands:
     - pip check
-    - fastapi --help
```

Comment:
If the upstream `pyproject.toml` defines the entry point (https://github.com/fastapi/fastapi-cli/blob/main/pyproject.toml#L52), I think we should probably test it, unless there is a reason not to.